### ISSUE TITLE
how to trigger an event on focus

### ISSUE THREAD

**Issue by binodstha** [srigar/multiselect-react-dropdown#227]

**Comment by manjesh-yadav** [comment]
Did you find any solution?

**Comment by rashmimhatre100** [comment]
Did anyone find solution? I am asking because from accessibility point of view. Dropdown should be opening by default. It should be only open if spacebar or enter key is pressed.
